app/models/manageiq/providers/ansible_tower/automation_manager/machine_credential.rb

```diff
@@ -1,2 +1,85 @@
 class ManageIQ::Providers::AnsibleTower::AutomationManager::MachineCredential < ManageIQ::Providers::AutomationManager::Authentication
+  def self.fields
+    return {
+      :description => {
+        :type       => 'string',
+        :required   => false,
+        :label      => 'Description',
+        :help_text  => 'Optional description of this credential.',
```

Comment:
So.. depends.. it's either `N_` or `_`, the first if you want the string to get in the translation catalog but don't want to translate it right away (meaning you have to pass it through `_` at some later point), the latter if you want it translated right away.

Right now, `help_text` is never used anywhere in the UI, so .. no idea which :).